driver/pace/driver/run.py

```diff
@@ -363,6 +378,8 @@ class DriverConfig:
     layout: Tuple[int, int]
     dt_atmos: float
     diagnostics_config: DiagnosticsConfig
+    performance_config: PerformanceConfig
+    experiment_name: str = "test"
```

Comment:
Should experiment_name and git_hash be part of PerformanceConfig? They aren't used otherwise in Driver.

Is it possible to programmatically determine the git hash inside Python, instead of passing it as a yaml/command-line configuration value?

Reply:
Moved `experiment_name` to `PerformanceConfig`. `git_hash` is determined from within python;